config/webpack.config.js

```diff
@@ -578,8 +575,8 @@ module.exports = function(webpackEnv) {
         new MiniCssExtractPlugin({
           // Options similar to the same options in webpackOptions.output
           // both options are optional
```

Comment:
so this is cache busting for css.  just want to make sure that we want to do this.

Reply:
Looking at CTS this is what we do now so this is probably ok.  Just need to make sure to remember to clear caches first when things look wonky